demonstrations/tutorial_falqon.py

```diff
@@ -0,0 +1,477 @@
+r"""
+Feedback-Based Quantum Optimization (FALQON)
+============================================
+
+.. meta::
+    :property="og:description": Solve combinatorial optimization problems without a classical optimizer
+    :property="og:image": https://pennylane.ai/qml/_images/falqon_thumbnail.png
+
+.. related::
+
+   tutorial_qaoa_intro Intro to QAOA
+   tutorial_qaoa_maxcut QAOA for MaxCut
+
+*Authors: David Wakeham and Jack Ceroni. Posted: XXX. Last updated: XXX.*
+
+-----------------------------
+
+While the
+`Quantum Approximate Optimization Algorithm (QAOA) <https://pennylane.ai/qml/demos/tutorial_qaoa_intro.html>`__
+is one of the best-known processes for solving combinatorial optimization problems with quantum computers,
+it has drawbacks: convergence isn't guaranteed, as the optimization procedure can become "stuck" in local minima.
+
+.. figure:: ../demonstrations/falqon/global_min_graph.png
+    :align: center
+    :width: 70%
+
+In this demo, we'll be implementing the FALQON algorithm: a feedback-based algorithm
+for quantum optimization, introduced in `this paper <https://arxiv.org/pdf/2103.08619.pdf>`__.
+It is similar in spirit to QAOA, but instead uses iterative feedback steps rather than a global optimization
+over parameters. We will show how to implement FALQON in PennyLane and test its performance on the MaxClique
+problem in graph theory.
+
+Theory
+------
+
+To solve combinatorial optimization problems using a quantum computer, a typical strategy is to encode
+the solution to the problem as the ground state of *cost Hamiltonian* :math:`H_c`, and then use some procedure to drive
+the system from an initial state into the ground state of :math:`H_c`. FALQON falls under this broad scheme.
+
+Consider a quantum system governed by a Hamiltonian of the form :math:`H = H_c + \beta(t) H_d`. These kinds of
+Hamiltonians appear often in the theory of `quantum control <https://quantiki.org/wiki/quantum-control-theory>`__: a
+field of inquiry which studies how a quantum system can be driven from one state to another.
+The choice of :math:`\beta(t)` allows us to decide which state we
+want a system governed by such a Hamiltonian to evolve towards.
+
+The time-dependent Schrodinger equation tells us that the dynamics of the system are given by:
+
+.. math:: i \frac{d}{dt} |\psi(t)\rangle = (H_c + \beta(t) H_d) |\psi(t)\rangle,
+
+where we set :math:`\hbar = 1`. Now suppose our objective is to drive the system
+to the state :math:`|\psi\rangle`: the ground state of :math:`H_c`. Phrased differently, we would like to minimize
```

Comment:
```suggestion
to the ground state of :math:`H_c`, which we denote by :math:`|\psi\rangle`. Phrased differently, we would like to minimize
```

Just to emphasize what the state conceptually is before giving it a label.